src/server.js

```diff
@@ -9,7 +9,7 @@ import createHistory from 'react-router/lib/createMemoryHistory';
 import { Provider } from 'react-redux';
 import { IntlProvider } from 'react-intl';
 import cookie from 'react-cookie';
-import raven from 'raven';
+import Raven from 'raven';
```

Comment:
sorry, this is probably unrelated to the PR. Happy to create a new one. But it's scoped to this file.